subt/serialize.py

```diff
@@ -0,0 +1,27 @@
+import io
+import numpy as np
+
+from osgar.bus import BusShutdownException
+from osgar.node import Node
+
+def serialize(arr):
```

Comment:
Maybe, here could be also some "deserialize" function/option?

Reply:
There is no need to do anything with numpy arrays as they are supported in osgar as-is. Just publish them directly.

Reply:
The reason for this is a possibility to send numpy array through zmq.

Reply:
Just use osgar.lib.serialize.

Reply:
osgar.lib.serialize calls `msgpack.packb` [at the end](https://github.com/robotika/osgar/blob/8928279c06222923c64efcfa58b4dc00c28162e6/osgar/lib/serialize.py#L26) and so does [ZMQ push.py](https://github.com/robotika/osgar/blob/8928279c06222923c64efcfa58b4dc00c28162e6/subt/push.py#L40). Together it is one message pack too many.

Reply:
It is never used together as `osgar.lib.serialize` is called only when writing to logs by default. I see `push.py` is not calling it, which I would consider a bug as `pull.py` does use it. If push.py called serialize instead of plain msgpack.packb, this node would not have been needed.

https://github.com/robotika/osgar/blob/8928279c06222923c64efcfa58b4dc00c28162e6/subt/pull.py#L36-L38

Reply:
This sounds cleanest.